File: in-motion-server-java/src/main/java/com/inmotion/inmotionserverjava/model/message/UpdatePostVideoMetadataMessage.java
package com.inmotion.inmotionserverjava.model.message;

import java.io.Serializable;
import java.util.List;

public record UpdatePostVideoMetadataMessage(
        String postId,
        String authorId,
        List<VideoMetadataMessage> videosMetaData
) implements Serializable {
}

File: in-motion-server/Shared/IMS.Shared.Messaging/Messages/PostVideos/UpdatePostVideoMetadataMessage.cs
namespace IMS.Shared.Messaging.Messages.PostVideos;

public class UpdatePostVideoMetadataMessage
{
    public string PostId { get; set; }
    
    public string AuthorId { get; set; }
    
    public IEnumerable<VideoMetaDataMessage> VideosMetaData { get; set; }
}